src/events/message/index.ts

```diff
@@ -29,13 +30,15 @@ export default class MessageEvent extends Event {
 				message.content.trim().toLowerCase(),
 			)
 		) {
-			const pingMessage = await message.channel.send("Ping ?").catch(noop);
+			const pingMessage = await message.channel
+				.send(`Ping ? (Cela peut durer jusqu'à ${REQUEST_TIMEOUT / 1000}s)`)
```

Comment:
plus haut : `const requestTimeout = Math.round(REQUEST_TIMEOUT / 1000);`
```suggestion
				.send(`Ping ? *Cela peut durer jusqu'à ${requestTimeout} secondes*`)
```

Reply:
étant donné qu'on fonctionne déjà avec des ms, on aura grand max 3 chiffres après la virgule, ce qui est pas non plus dérangeant

Reply:
```.send(`Ping ? *Cela peut durer jusqu'à ${REQUEST_TIMEOUT / 1000} secondes*`)``` au moins